opennms-webapp/src/main/webapp/WEB-INF/jsp/graph/results.jsp

```diff
@@ -198,6 +199,23 @@
             </c:if>
         </h3>
      </div> <!-- panel-heading -->
+       <c:if test="${fn:contains(resultSet.resource.resourceType.label, 'SNMP Interface')}">
+         <c:if test="${fn:contains(resultSet.resource.label,'(*)') != true}">
+           <c:forEach var="attribute" items="${resultSet.resource.attributes}">
+             <c:if test="${fn:contains(attribute.name, 'ifIndex')}">
+               <c:set var="ifIndex" value="${attribute.value}"/>
+             </c:if>
+             <c:if test="${fn:contains(attribute.name, 'nodeId')}">
+               <c:set var="nodeId" value="${attribute.value}"/>
+             </c:if>
+           </c:forEach>
+           <div ng-app="onms-ksc" ng-controller="checkFlowsCtrl" ng-init="getFlowCount(${nodeId}, ${ifIndex}, ${results.start.time}, ${results.end.time})">
+               <div ng-show="hasFlows">
+                 <a target=_blank" href="http://grafana:3000/dashboard/var-node=${nodeId}&amp;var-interface=${ifIndex}" style="padding-right: 3px" title="Open Flows for ${resultSet.resource.label}"><button type="button" class="btn btn-default btn-xs"><i class="fa fa-bar-chart" aria-hidden="true"></i></span></button></a>
```

Comment:
We should make this URL configurable, allowing the user to provide some string with placeholders such as $nodeId, $ifIndex, $start and $end, and replace those tokens as necessary.

Ideally, we would do this server side, instead of in the JSP here. Perhaps introduce a new endpoint in the flow API?

Reply:
Even if this link is a *resource level* link, I think it might be better to add the icon alongside the existing icons at the top of the graphs. We can do this for each graph in the resource.